atlassian/rest_client.py

```diff
@@ -60,7 +60,46 @@ def __init__(
         cloud=False,
         proxies=None,
         token=None,
+        backoff_and_retry=False,
+        retry_error_matches=[(429, "Too Many Requests"),
+                                         (429, "Unknown Status Code")],
```

Comment:
503 also deserve a retry IMO, since jira cloud does throw it from time to time

posting it here if someone want to update this PR or create a new one using requests.Session retry mechanism

Reply:
I included 503 in my new PR #1339 since that is in the default list of urllib3.util.Retry